tech/languages/csharp/csharp-ide.md

```diff
@@ -6,45 +6,43 @@ order: 2
 
-_.NET Core and Mono_
+_1. Download and install using direct link to [Flatpak](/deployment/Flatpak/about.html)_
 
-JetBrains Rider is full featured IDE for all things C#, and even F# on .NET Core. If you are familiar with other JetBrains products, such as the IntelliJ or Resharper, you might like this one as well.
+## Installing Eclipse
 
-[jetbrains.com/rider](http://jetbrains.com/rider)
-
-## Visual Studio Code
-
-_.NET Core Only_
-
-Visual Studio Code (aka VS Code) is plugin-based IDE that by itself won't just run C#. You have to install the C# plugin, and NuGet Package Manager.
-
-Download the Visual Studio Code at [code.visualstudio.com](https://code.visualstudio.com) and the C# Extension at [marketplace.visualstudio.com](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp)
-
-## Eclipse
-
-_.NET Core and Mono_
+First install Eclipse and Eclipse Marketplace
+```
+$ sudo dnf install eclipse eclipse-mpc
+```
+then install the aCute extension from the [Marketplace](https://marketplace.eclipse.org/content/acute-c-edition-eclipse-ide-experimental)
 
-You can use the [aCute plugin](https://marketplace.eclipse.org/content/acute-c-edition-eclipse-ide-experimental) on top of the established and popular [Eclipse IDE](/tools/eclipse/about.html). This C# extension for Eclipse IDE relies on [OmniSharp](http://www.omnisharp.net) and [Language Server Protocol](https://github.com/Microsoft/language-server-protocol). It comes with complete edition features for C# in the Eclipse IDE, relying on the Eclipse IDE provides all typical features of a good IDE (Polyglot development, SCM, deployment...) and give access to a rich ecosystem of extensions for about all existing technologies.
+## Installing MonoDevelop
 
 ```
-$ sudo dnf install eclipse eclipse-mpc
+$ sudo dnf install monodevelop
 ```
-then install the aCute extension from [Eclipse Marketplace](https://marketplace.eclipse.org/content/acute-c-edition-eclipse-ide-experimental)
 
-## MonoDevelop
+## Installing JetBrains Rider
 
-_Mono only_
+* Apply for a [license for Open Source projects or students](https://www.jetbrains.com/store/?fromMenu#edition=discounts) and download Rider at [jetbrains.com/rider](http://jetbrains.com/rider)
+* Download the [Early Access](https://www.jetbrains.com/rider/eap) version
 
-MonoDevelop provides the best C# experience on Mono, the open-source multiplatform implementation of the .NET Framework.
+## Installing Visual Studio Code
 
-[monodevelop.com](http://www.monodevelop.com)
+Download the Visual Studio Code at [code.visualstudio.com](https://code.visualstudio.com) and the C# Extension from the [Marketplace](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp)
```

Comment:
Note: I think there are too many `the` articles and I think the sentence could work all without them. 

*[ non-blocking ]*

Reply:
Sorry- I did forget to mention this before- please replace also the `code.visualstudio.com` to some non-url word, like f.e. `VS Code website`.

Reply:
@aslicerh another English check please :D